peps/pep-0736.rst

```diff
@@ -161,18 +196,22 @@ to compute:
 * The number of lines of code which could be saved by using this syntactic sugar
   to reduce the need for line wraps.
 
-===================================================================== ================ ============== =============== =====================
-Statistic                                                             `cpython <a_>`__ `numpy <b_>`__ `pandas <c_>`__ `scikit-learn <d_>`__
-===================================================================== ================ ============== =============== =====================
-Number of keyword arguments of the form ``f(x=x)`` at invocation      4,225            2,768          13,235          8,342
-Percentage of keyword arguments of the form ``f(x=x)`` at invocation  11.06%           13.17%         17.24%          18.64%
-Lines saved                                                           290              247            935             794
-===================================================================== ================ ============== =============== =====================
+The purpose of this exercise was to compute statistics about the prevalence of
+this pattern and should not be interpreted as a recommendation that the proposed
+syntactic sugar should be applied universally.
+
+===================================================================== =============== ================ ============== ==============
+Statistic                                                             `polars <a_>`__ `fastapi <b_>`__ `rich <c_>`__  `httpx <d_>`__
+===================================================================== =============== ================ ============== ==============
+Number of keyword arguments of the form ``f(x=x)`` at invocation      1,654           1,408            566            759
+Percentage of keyword arguments of the form ``f(x=x)`` at invocation  15.83%          28.11%           15.74%         45.13%
+Lines saved                                                           170             35               62             117
+===================================================================== =============== ================ ============== ==============
 
-.. _a: https://github.com/python/cpython/pull/111423/
-.. _b: https://github.com/numpy/numpy/pull/25021/
-.. _c: https://github.com/pandas-dev/pandas/pull/55744/
-.. _d: https://github.com/scikit-learn/scikit-learn/pull/27680/
+.. _a: https://github.com/joshuabambrick/polars/pull/1
+.. _b: https://github.com/joshuabambrick/fastapi/pull/1
+.. _c: https://github.com/joshuabambrick/rich/pull/1
+.. _d: https://github.com/joshuabambrick/httpx/pull/1
 
 Based on this, we note that  the ``f(x=x)`` keyword argument pattern is
 widespread, accounting for 10-20% of all keyword argument uses.
```

Comment:
not to self to update this based on the newer library set (15% to just around half)